andela/travel_tool_back
# #160751499 Configure commitplease to check commit messages adhere to playbook's standards.

#### What does this PR do?
configure commitplease to make sure every commit message is per the engineering playbook standard.

#### Description of Task to be completed?
When a commit message is created commitplease will check if the PR is per the playbooks standard through it's inbuilt git pre-commit hook.If not, there will be an error message prompted on the CLI giving details about the commit message.

#### How should this be manually tested?
- After cloning the repo, ```git checkout``` into ```ch-add-commitplease-160751499```
- install dependencies ```yarn install```
- Make a small change and commit with a commit message that doesn't adhere to the playbook's standards.There should be and error message on the CLI detailing this.

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[#160751499](https://www.pivotaltracker.com/story/show/160751499)

#### Screenshots (if appropriate)
<img width="918" alt="screen shot 2018-11-05 at 14 08 42" src="https://user-images.githubusercontent.com/19623571/47994837-932b3000-e104-11e8-8f32-ddf2f66db9fd.png">
#### Questions:
